docs/layouts.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Blazor layouts
+author: rstropek
+description: Learn how to create reusable layout components for Blazor apps.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/09/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/layouts
+---
+# Blazor layouts
+
+By [Rainer Stropek](https://www.timecockpit.com)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+Blazor apps typically contain more than one page. Layout elements, such as menus, copyright messages, and logos, must be present on all pages. Copying the code of these layout elements onto all of the pages of an app isn't an efficient solution. Such duplication is hard to maintain and probably leads to inconsistent content over time. *Blazor layouts* solve this problem.
+
+## What are Blazor layouts?
+
+Technically, a layout is just another Blazor component. A layout is defined in a Razor template or in C# code and can contain data binding, dependency injection, and other ordinary features of components. Two additional aspects are special and turn a Blazor component into a Blazor layout:
+
+* The layout component must implement `Microsoft.AspNetCore.Blazor.Layouts.ILayoutComponent`. This interface adds a property `Body` to the component, which contains the content to be rendered inside the layout.
```

Comment:
@guardrex can we make this an API ref link?

Reply:
Yes, it should work out to something like (assuming `www`) ...

https://www.blazor.net/api/Microsoft.AspNetCore.Blazor.Layouts.ILayoutComponent.html

For reference, the current live link right now is ...

https://aspnet.github.io/Blazor.Docs/api/Microsoft.AspNetCore.Blazor.Layouts.ILayoutComponent.html

Reply:
@guardrex Should I use the *www.blazor.com* or the *aspnet.github.io" link? Should I turn all Blazor type references into API ref links?

Reply:
Use this one ...

```
https://www.blazor.net/api/Microsoft.AspNetCore.Blazor.Layouts.ILayoutComponent.html
```
... and I'll fix it later if it needs to change.